libs/analytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -2273,8 +2273,6 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "app_settings_privacy_settings_tapped";
             case APP_SETTINGS_OPEN_DEVICE_SETTINGS_TAPPED:
                 return "app_settings_open_device_settings_tapped";
-            case APP_SETTINGS_OPTIMIZE_IMAGES_POPUP_TAPPED:
-                return "app_settings_optimize_images_popup_tapped";
```

Comment:
I was wondering if we should deprecate this event 🤔. I haven't found documentation about it, so I presume that it will be automatically marked as deprecated when we stop receiving it.

Reply:
Ah, good thought! I went digging and found these steps for deprecating events, which I'll follow for all the popup-related events: pabtAt-Xk-p2#comment-1444